spec/mailers/previews/provider_mailer_preview.rb

```diff
@@ -120,10 +120,12 @@ def find_service_is_now_open
   end
 
   def set_up_organisation_permissions
```

Comment:
maybe we can have two views to show both variations?

Reply:
Yeah I agree it's a bit messy atm

Reply:
https://github.com/DFE-Digital/apply-for-teacher-training/commit/4a477dc867d32eda76d91e67538c881fe0f47ba0